http-verbs-common/src/test/scala/uk/gov/hmrc/http/HttpGetSpec.scala

```diff
@@ -177,7 +177,7 @@ class HttpGetSpec
 
     "return a url with encoded param pairs" in {
       val expected =
-        Some("http://test.net?email=test%2Balias%40email.com&data=%7B%22message%22%3A%22in+json+format%22%7D")
+        Some("http://test.net?email=test%2Balias@email.com&data=%7B%22message%22:%22in+json+format%22%7D")
```

Comment:
As per  RFC3986 - `@` and `:` can be safely used in query params. 
Discussed here - https://github.com/softwaremill/sttp-model/issues/11